input/script_utils.go

```diff
@@ -2201,7 +2201,8 @@ func NewRemoteCommitScriptTree(remoteKey *btcec.PublicKey,
 	// their outputs.
 	builder := txscript.NewScriptBuilder()
 	builder.AddData(schnorr.SerializePubKey(remoteKey))
-	builder.AddOp(txscript.OP_CHECKSIG)
+	builder.AddOp(txscript.OP_CHECKSIGVERIFY)
```

Comment:
Shouldn't it be:

```
builder.AddOp(txscript.OP_CHECKSIG)
builder.AddOp(txscript.OP_1)
builder.AddOp(txscript.OP_CHECKSEQUENCEVERIFY)
builder.AddOp(txscript.OP_DROP)
```

According to the latest version of https://github.com/Roasbeef/lightning-rfc/blob/0ed59d1edff5066b0f23f106536937447f7739e6/bolt-simple-taproot.md#to-remote-outputs?

Reply:
hmm...weird it's not caught in the unit test

Reply:
Ah yeh I diverged here slightly.

@yyforyongyu either version works: the one I have rn will make it fail early (verify removes the item from the stack), while the other one will fail only after the 0 is left on the stack